src/components/user-list.html

```diff
@@ -1,24 +1,75 @@
-<link rel="import" href="./user-profile.html">
+<link rel="import" href="../../node_modules/@banno/polymer/polymer.js">
+<link rel="import" href="../../node_modules/@banno/polymer/lib/elements/dom-repeat.js">
 <link rel="import" href="./user-profile.html">
 
 <dom-module id="user-list">
-    
-    <template>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
-        <user-profile></user-profile>
+  <style>
+   /* Add Styles , not taking so far???*/
+   .centered{
+     text-align: center;
+   }
+   
+   .button {
+        background-color: #87CEFA;
+        border: none;
+        color: white;
+        padding: 5px 12px;
+        text-align: center;
+        text-decoration: none;
+        display: block;
+        margin: auto;
+        font-size: 16px;
+        margin-top: 25px;
+      }
+
+      button:hover {
+        background-color: rgb(196, 226, 245);
+      }
+  </style>
+  
+  <template>
+    <div class="centered">
+      <h2 class="centered">User List:</h2>
+      <button class="button" value="Refresh Page" onClick="window.location.reload();">Generate User List</button>
```

Comment:
![... but why?](http://giphygifs.s3.amazonaws.com/media/1M9fmo1WAFVK0/giphy.gif)

shouldn't need to have the window reload just to get updated data.